includes/class-wc-siftscience-events.php

```diff
@@ -440,8 +448,8 @@ private function send_order_event( $order_id, $type = 'create' ) {
 		 * @param string $order_id The order ID.
 		 */
 		public function update_order_status( $order_id ) {
-			$order_id = wc_get_order( $order_id );
-			$user_id  = $this->options->get_user_id( $order_id );
+			$order = wc_get_order( $order_id );
+			$this->order_status->try_update_order_status( $order, $this->options->get_user_id( $order ) );
```

Comment:
![image](https://user-images.githubusercontent.com/11143071/119378844-90b56500-bc73-11eb-97be-6c669c2ce5cb.png)

The function try_update_order_status() does not take a user-id as its second parameter.

Reply:
I didn't catch that ok